### ISSUE TITLE
package-info.java not parsed/supported

### ISSUE THREAD

**Issue by markphilpot** [thorbenvh8/prettier-java#43]
```
[error] ~~snip~~/package-info.java: Error: Sad sad panda, parsing errors detected in line: 2, column: 1!
[error] Expecting: one of these possible Token sequences:
[error]   1. ['class']
[error]   2. ['enum']
[error]   3. ['interface']
[error]   4. ['@']
[error] but found: 'package'
[error]     at Object.parse (~~node_modules/chevrotain-java/src/index.js:20:11)
[error]     at Object.parse (~~node_modules/prettier-plugin-java/src/parser.js:6:25)
[error]     at Object.parse$3 [as parse] (~~node_modules/prettier/bin-prettier.js:23712:19)
[error]     at formatWithCursor (~~node_modules/prettier/bin-prettier.js:29396:27)
~~snip~~
```

Example contents:

```
@MyImmutableStyle
package com.mypackage;
```

**Comment by thorbenvh8** [comment]
Thanks for the issue :)
How many package annotation are possible "1" or "*"?

**Comment by markphilpot** [comment]
Fairly certain is "*" (zero or more)

**Comment by thorbenvh8** [comment]
Have a solution for zero or 1 in the chevrotain-parser, there needs to be some refactoring. Will do this after the other little tickets are closed :)

**Comment by thorbenvh8** [comment]
Fixed in 0.0.10!
Thanks for the raising the issue! 👍
